tcf_website/models/models.py

```diff
@@ -290,16 +290,6 @@ def is_recent(self):
         """Returns True if course was taught in current semester."""
         return self.semester_last_taught == Semester.latest()
 
-    def average_rating(self):
-        """Compute average rating."""
-        return Review.objects.filter(course=self).aggregate(
-            models.Avg('recommendability'))['recommendability__avg']
-
-    def average_difficulty(self):
-        """Compute average difficulty score."""
-        return Review.objects.filter(course=self).aggregate(
-            models.Avg('difficulty'))['difficulty__avg']
-
```

Comment:
Are you sure we should remove these? I assume they're meant to be replaced by API functionality, but I imagine there's some possibility that we may need them again for something in the future, and it might be easier/convenient to develop using this normal Django way if it's possible. If the API is strictly better/faster, then it makes more sense.

Reply:
Done in 1d04b3e2c44cb0af4b7ee98cb41da7793d7bcfb9